_posts/2015-04-08-rfp.md

```diff
@@ -0,0 +1,315 @@
+
+**Sealed proposals (three hard copies and one digital copy) must be received no later than noon May 8, 2015. Postmarks are not sufficient. Proposals by fax or email are not acceptable**.  
+
+Any questions or Requests for Information (RFI) related to the proposal must be received in writing by April 17, 2015.  All questions will be documented: both questions and answers will be available to the public online at www.pittsburghpa.gov/Citiparks by April 22, 2015.  
+
+Contact person for all inquiries and for submission of proposals:  
+
+
+Ms. Louann Horan, Manager of Operations and Administration  
+Department of Parks & Recreation  
+City of Pittsburgh  
+Fifth Floor, City-County Building, Room 459  
+414 Grant Street  
+Pittsburgh, PA 15219  
+412 255-2362  
+Louann.Horan@PittsburghPA.gov
+
+####**SECTION 1. BACKGROUND**
+
+The Pittsburgh Department of Parks & Recreation (Citiparks) is seeking proposals to provide custodial and cleaning service at 22 facilities, including ten (10) recreation centers and twelve (12) Healthy Active Living (senior) centers.  
+
+Citiparks is the department within the City of Pittsburgh that provides parks programming for children, adults and seniors.  In addition to recreational instruction, Citiparks’ provides wellness, fitness, cultural, art and educational enrichment for city residents as well as conducting a wide variety of special events for families and communities throughout the city.  
+
+This Request for Proposals specifically addresses the City of Pittsburgh’s need for regular, efficient and effective custodial and cleaning services for the 22 recreation and Healthy Active Living (senior) centers specified in this document.  These facilities, ranging from 2,000 – 15,000 sq. ft. have basic custodial needs that must be addressed multiple times each week by a custodial contractor.
+
+Regular custodial duties include, but are not limited to:
+
+* Floor mopping sweeping
+
+* Carpet vacuuming
+
+* Equipment and furnishing dusting
+
+* Trash collection mitigation, including collection and removal
+
+* Disinfecting of equipment, kitchens, eating areas and restrooms
+
+* Window and mirror washing
+
+* Litter picking
+
+* Dispenser replenishing
+
+* Wipe down of desks, tables, equipment and furniture
+
+* Light bulb replacement 
+
+* Other light duties, as assigned.
+
+
+In lieu of an established fee, Citiparks is seeking proposals that provide an hourly rate for providing the services above for as few as four specific facilities and as many as all 22 facilities.  Respondents may group facilities in any manner that they deem most appropriate and may also provide multiple proposals based on varying grouping patterns or number of facilities to be served.
+
+Respondents should be aware that Citiparks would anticipate approximately six (6) hours of custodial service per week will be required, on average, not including travel per facility
+
+####**SECTION 2. REQUIREMENTS**
+
+####**A. General**
```

Comment:
There should probably be some way to differentiate between the larger sections (SECTION 1) and the subsections (A. GENERAL). Right now they are both bold h4 elements.